flexget/plugins/clients/flood.py

```diff
@@ -0,0 +1,131 @@
+import os
+from loguru import logger
+from requests import Session
+from requests.exceptions import RequestException
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.template import RenderError
+from os import error
+from requests import Session
+from loguru import logger
+import json
+import base64
+
+logger = logger.bind(name='flood')
+
+class OutputFlood:
+    schema = {
+       'type': 'object',
+        'properties': {
+            'host': {'type': 'string'},
+            'port': {'type': 'integer'},
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'directory': {'type': 'string'},
+            'tags': {'type': 'array'},
+        },
+        'additionalProperties': False,
+    }
+
+    def __init__(self):
+        self.session = Session()
+        self.connected = False
```

Comment:
It would be best not to have anything in the plugin init. This will cause creating unnecessary Sesson() objects even if flood plugin is never used. (Not that it's a huge issue, but still).

Global variables aren't good either. Can session expire between task executions, how does connected flag work in that case?